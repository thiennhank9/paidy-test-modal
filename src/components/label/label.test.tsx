import TestRenderer from "react-test-renderer";
import Label from "./label";

describe("Label", () => {
  test("render Label", () => {
    const testRenderer = TestRenderer.create(<Label />);
    expect(testRenderer.toJSON()).toMatchSnapshot();
    const testInstance = testRenderer.root;
    const label = testInstance.findByType("label");
    expect(label).toBeDefined();
  });
});
