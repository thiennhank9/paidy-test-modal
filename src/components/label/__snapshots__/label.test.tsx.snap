// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label render Label 1`] = `<label />`;
